templates/vertical-full-page-map/page-setup.js

```diff
@@ -7,5 +7,25 @@ if (window.locatorBundleLoaded) {
   loadFullPageMap();
 } else {
   const locatorBundleScript = document.querySelector('script#js-answersLocatorBundleScript');
-  locatorBundleScript.onload = loadFullPageMap;
+  locatorBundleScript.onload = () => {
+    window.locatorBundleLoaded = true;
+    locatorBundleScript.dispatchEvent(new Event('locator-bundle-loaded'));
+    loadFullPageMap();
+  }
 }
+
+/**
+ * Registers listeners on the card once the locator bundle is loaded
+ *
+ * @param {ANSWERS.Component} card A location card
+ */
+function registerVerticalFullPageMapCardListeners(card) {
+  if (window.locatorBundleLoaded) {
+    new VerticalFullPageMap.CardListenerAssigner({card: card}).addListenersToCard();
```

Comment:
Would it be possible to move CardListenerAssigner into HitchhikerJS? I know in terms of bundle size it's not optimal, but I think it could be much simpler? Long term I think we would want to get the cards into per page bundles, instead of inlining them, and tree shake the theme (possibly using the cards as entry points)

Reply:
oh I think Tom's suggestion also makes sense! I was worried that flipping the script order would cause the search on load to be delayed, but
1. not sure if that's the case and
2. even if it is, we may prefer that over moving the CardListenerAssigner to the HitchhikerJS and pushing it onto other pages.

Reply:
Ya, I think that would work. There isn't much code in the CardListenerAssigner, so the impact on the HH bundle would be minimal. Since Tom is good to move forward with this solution, I'll make an item for investigating that approach in the future. That might resolve the issue where the map loads while being zoomed in.